### PR TITLE
Add CloudWatch configuration file

### PR DESCRIPTION
**Pull request by tschaffter**
Partially address #30 

Add CW agent configuration file used on EC2 instances (data node, controller, elk).

This configuration file was generated using `sudo /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-config-wizard`. The options selected are the mostly the ones listed in the article [Cloudwatch Agent Manual Install](https://www.wellarchitectedlabs.com/cost/200_labs/200_aws_resource_optimization/4_memory_plugin/).

@thomasyu888 There's probably a better place for this file than the root directory of this GH repo. Let me know if you want me to place it somewhere else.

### CODE REVIEW

**Comment by thomasyu888** [comment]
@tschaffter Can you add the description you posted to the readme so that we have record of how to set up cloudwatch.

**Comment by tschaffter** [comment]
@thomasyu888 I added the instructions on how to install and run CW to the README.

**Comment by tschaffter** [comment]
@thomasyu888 Why is the test failing? Do you know how to fix the issue?

File: README.md [hunk 1]
```diff
@@ -17,3 +17,48 @@ pre-commit run --all-files
...
+    sudo /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -m ec2 -a status
+    ```
+
+To stop the agent:
+
+```
+sudo /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -m ec2 -a stop
+```
+
+[Download the CloudWatch agent for Ubuntu (x86-64)]: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/download-cloudwatch-agent-commandline.html
```
Comment (thomasyu888) [inline README.md:64]:
```suggestion
[Download the CloudWatch agent for Ubuntu (x86-64)]: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/download-cloudwatch-agent-commandline.html

```

**Review by thomasyu888 — CHANGES_REQUESTED** [review]

**Review by thomasyu888 — APPROVED** [review]

**Pull request merged**